regolith/helpers/l_contactshelper.py

```diff
@@ -0,0 +1,136 @@
+"""Helper for finding and listing contacts from the contacts.yml database.
+Prints name, institution, and email (if applicable) of the contact.
+"""
+import datetime 
+import dateutil
+import dateutil.parser as date_parser
+from dateutil.relativedelta import relativedelta
+import sys
+
+
+from regolith.helpers.basehelper import SoutHelperBase
+from regolith.fsclient import _id_key
+from regolith.tools import (
+    all_docs_from_collection,
+    get_pi_id,
+    fuzzy_retrieval,
+)
+
+TARGET_COLL = "contacts"
+HELPER_TARGET = "l_contacts"
+
+def subparser(subpi):
+    subpi.add_argument("-n", "--name", help='finds contacts based off a list of names or name fragments', nargs = '+')
+    subpi.add_argument("-i", "--inst", help='finds contacts based off an institution or an institution fragment')
+    subpi.add_argument("-d", "--date", help='finds contacts based off a date in ISO format (YYYY-MM-DD) with a default range of 4 months centered around the date (d +/- 2 month)')
+    subpi.add_argument("-r", "--range", help='optional argument that sets range (in months) centered around date d (d +/- r/2)')
+    subpi.add_argument("-no", "--notes", help='finds contacts based off notes or miscellaneous information about the contact', nargs = '+')
```

Comment:
this breaks CLI logic so you will have to do something like ``-o, --notes``.   Reuse the language of the name help message.  I think the problem I have with the current formulation of hte help is that I have to read all the way to the end to find out what I am supposed to be entering.  Of course I am running the "list contact" helper so I kind of know that it is going to "find a contact based on <something>"